src/main/java/com/GotYourBac/gotYourBac/config/WebSecurityConfig.java

```diff
@@ -40,12 +40,12 @@ protected void configure(HttpSecurity http) throws Exception {
                 .cors().disable()
                 .csrf().disable()
                 .authorizeRequests()
-                .antMatchers("/", "/css/**", "/register", "/login").permitAll()
-                .antMatchers(HttpMethod.POST, "/register").permitAll()
+                .antMatchers("/", "/css/**", "/registration", "/error").permitAll()
```

Comment:
you should change this antMatchers to only be for get requests with the first argument of HttpMethod.GET

Reply:
added Httpmethod.get for antMatchers